src/shared/utils/exportData/getJourneyCSVObject.test.ts

```diff
@@ -169,12 +169,72 @@ const result = {
   input_user_nickname: 'Mock input user',
   version: '2.0.0',
   event_id: null,
-  timezone_offset: '',
-  legacy_user_id: '',
+  timezone_offset: null,
+  legacy_user_id: null,
+};
+
+const newResult = {
```

Comment:
A bit of a nit, but let's just call this `result` as it is the result when the flag is on and also when it is ultimately archived. Minimizes cleanup when flag is removed.